python-client/giskard/models/utils.py

```diff
@@ -10,3 +12,12 @@ def map_to_tuples(data: Iterator):
         return data
 
     return map(lambda x: (x,), data)
+
+
+def numpy_encoder(some_list: List):
```

Comment:
I'd rename it to something less cryptic like `np_types_to_native` or something similar to give a hint of what it does